extensions/wikia/AdEngine/js/run/desktop.run.js

```diff
@@ -86,6 +88,8 @@ require([
 			sourcePoint.initDetection();
 		}
 
+		pageFair.initDetection();
```

Comment:
I'm not sure how we want to do it for PF, because for SP when we run initDetection equals run detection script. But for PF detection script is run from an external file which calls a function in our detection module. Anyway maybe `if` based on the flag is a safer solution than current.